mythril/src/acpi/rsdt.rs

```diff
@@ -371,16 +371,21 @@ impl<'a, T: Array<Item = u8>> RSDTBuilder<'a, T> {
         })?;
 
         for (i, (name, (sdt, size))) in self.map.iter().enumerate() {
-            let table_name = format!("etc/mythril/{}", str::from_utf8(name)?);
+            const LEN_OF_ETC_MYTHRIL :usize = 12;
```

Comment:
I think this can be refactored. We can at least write something like:

```
            const ETC_MYTHRIL: &'static str = "etc/mythril/";
            const LEN_OF_NAME: usize = 4;
            let mut table_name_bytes = [0u8; ETC_MYTHRIL.len() + LEN_OF_NAME];
            table_name_bytes[0..ETC_MYTHRIL.len()]
                .copy_from_slice(ETC_MYTHRIL.as_bytes());
            table_name_bytes[ETC_MYTHRIL.len()..].copy_from_slice(name);
            let table_name = str::from_utf8(&table_name_bytes)?;
```

But I'd actually like the constants pulled out given that the size, at least, is used elsewhere and must remain aligned with this.

Reply:
agreed.